modules/installation-initializing.adoc

```diff
@@ -150,20 +150,6 @@ endif::osp[]
 ifdef::vsphere,vmc[]
 VMware vSphere.
 endif::vsphere,vmc[]
-ifdef::rhv[]
```

Comment:
I don't think you want to remove this ifdef/endif that contains the RHV name.

Reply:
+1. The RHV name here is part of the sentence that precedes the ifdef. It should stay. Pending SME approval, the two warnings can go from master, but they need to stay in 4.6 (and earlier)

Reply:
Thanks. Updated.